lib/lte_link_control/lte_lc.c

```diff
@@ -886,6 +886,39 @@ int lte_lc_edrx_req(bool enable)
 	return err;
 }
 
+int lte_lc_pdp_context_set(enum lte_pdp_type type, const char *apn,
+			bool ip4_addr_alloc, bool nslpi, bool securePCO)
+{
+	static const char * const pdp_type_lut[] = {
+		"IP", "IPV6", "IPV4V6"
+	};
+
+	if (apn == NULL || type > PDP_TYPE_IPV4V6) {
+		return -EINVAL;
+	}
+
+	snprintf(cgdcont, sizeof(cgdcont),
+			"AT+CGDCONT=0,\"%s\",\"%s\",0,0,0,%d,0,0,0,%d,%d,%d",
+			pdp_type_lut[type], apn,
+			ip4_addr_alloc, nslpi, securePCO);
+
+	return 0;
+}
+
+int lte_lc_pdn_auth_set(u8_t auth_prot,
+			const char *username, const char *password)
+{
+	if (username == NULL || password == null || auth_prot > 3) {
```

Comment:
s/null/NULL
Is `null` acutually defined somewhere, does it compile?

Reply:
Should be NULL. Not sure how that happened. The code did compile before I pushed it.